.github/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: JDK ${{ matrix.java_version }}
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        # TODO Add 9, 10, 11, 12, and 13 after Kotlin 1.3.60
+        java_version: [1.8]
+    steps:
+    - name: Checkout
+    - uses: actions/checkout@v2
+    - name: Install JDK
+      uses: actions/setup-java@v2
+      with:
+        java-version: ${{ matrix.java_version }}
+    - name: Gradle Wrapper Validation
+      uses: gradle/wrapper-validation-action@v1
+    - name: Configure Gradle
+      # Gradle configuration install deps 
+      run: ./gradlew help
+    - name: Spot check
+      run: ./gradlew spotlessCheck --stacktrace
+    - name: Build Project
+      run: ./gradlew assemble --stacktrace
+    - name : Testing 
+      run: ./gradlew test --stacktrace
+    - name: Final Checks
+      run: ./gradlew check --stacktrace
```

Comment:
[snapshots](https://github.com/uber/motif/blob/master/.travis.yml#L17) should be enabled for this repo